plugins/inputs/sqlserver/sqlserverqueries.go

```diff
@@ -247,6 +247,24 @@ SELECT
 EXEC sp_executesql @SqlStatement
 `
 
+const SQLServerForceEncryption = `
+SET DEADLOCK_PRIORITY -10;
```

Comment:
This query won't run, single quotes have to be escaped inside the string itself (`'` → `''` ), and the output variable has to be declared inside the statement itself in order to have the proper scope.
also it's missing the instance name, and you won't be able to distinguish between multiple servers without it
```
SET DEADLOCK_PRIORITY -10;
DECLARE @SqlStatement nvarchar(max)

SET @SqlStatement = N'
DECLARE @ForceEncryption INT

EXEC [master].[dbo].[xp_instance_regread]
       @rootkey = ''HKEY_LOCAL_MACHINE'',
       @key = ''SOFTWARE\Microsoft\Microsoft SQL Server\MSSQLServer\SuperSocketNetLib'',
       @value_name = ''ForceEncryption'',
       @value = @ForceEncryption OUTPUT;

SELECT @@SERVERNAME AS sql_instance, @ForceEncryption AS ForceEncryption'

EXEC sp_executesql @SqlStatement
```